cvat/apps/limit_manager/models.py

```diff
@@ -2,10 +2,43 @@
 #
 # SPDX-License-Identifier: MIT
 
+from enum import Enum
 from django.db import models
+from django.contrib.auth.models import User
+from cvat.apps.organizations.models import Organization
 
-import cvat.apps.limit_manager.core.limits as core
+class LimitationTypeChoice(str, Enum):
+    DEFAULT = 'default'
+    PAID = 'paid'
+    MANUALLY = 'manually'
 
+    @classmethod
+    def choices(cls):
+        return tuple((x.value, x.name) for x in cls)
 
-class Limits(core.Limits, models.TextChoices):
-    pass
+    def __str__(self):
+        return self.value
+
+# TO-DO:
+#  - add validation user__isnull ^ org__isnull
+class Limitation(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
+    org = models.ForeignKey(Organization, on_delete=models.CASCADE, null=True)
+    type = models.CharField(max_length=32, choices=LimitationTypeChoice.choices(),
+        default=LimitationTypeChoice.DEFAULT)
+
+    # List of limitations:
+    #   - null means unlimited
+    #   - blank means that certain limit not applicable
+    memberships = models.PositiveIntegerField(null=True, blank=True)
```

Comment:
I'm not sure but probably one [JsonField](https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.JSONField) can be a better choice to keep all limits below.